source/_components/sensor.influxdb.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "InfluxDB Sensor"
+description: "Instructions how to integrate InfluxDB sensors within Home Assistant."
+date: 2016-10-26 23:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: influxdb.png
+ha_category: Sensor
+ha_release: 0.32
+---
+
+The `InfluxDB` sensor allows you to use values from an InfluxDB database to populate a sensor state.
+
+To configure this sensor, you need to define the sensor connection variables and a list of queries. A sensor will be created for each query.
+
+Configuration variables for the server:
+
+- **host** (*Optional*): IP address of your database host, eg. http://192.168.1.10. Defaults to `localhost`.
```

Comment:
Can you please do me a favor and check if that's right? We are talking about the IP address but show an URL. I guess that wrong in the InfluxDB component documentation as well.